src/Magnum/Platform/Sdl2Application.cpp

```diff
@@ -378,12 +378,11 @@ void Sdl2Application::mainLoop() {
                 event.type == SDL_MOUSEBUTTONDOWN ? mousePressEvent(e) : mouseReleaseEvent(e);
             } break;
 
-            case SDL_MOUSEWHEEL:
-                if(event.wheel.y != 0) {
-                    MouseEvent e(event.wheel.y > 0 ? MouseEvent::Button::WheelUp : MouseEvent::Button::WheelDown, {event.wheel.x, event.wheel.y}, 0);
-                    mousePressEvent(e);
-                } break;
-
+            case SDL_MOUSEWHEEL: {
+                MouseEvent e(event.wheel.y > 0 ? MouseEvent::Button::WheelUp : MouseEvent::Button::WheelDown, {event.wheel.x, event.wheel.y}, 0);
+                mousePressEvent(e);
```

Comment:
Yep, that's what I was referring to as a non-ideal API. :)

I would be much happier to get rid of the MouseWheel distinction completely and move towards defining a more generic "scroll" event.

As a side question, Is there any way to get the smooth/inertial scrolling that most native OSX applications have inside of Magnum?  I imagine it might require using a different platform than SDL2, and of course would not be cross-platform, but it would still be very nice to have.